src/BloomExe/InstallerSupport.cs

```diff
@@ -126,24 +126,43 @@ internal static void MakeBloomRegistryEntries()
 				return;
 			}
 			var installDir = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
+			if (installDir == null)
+				return;
 
+			var iconDir = Path.Combine(Path.Combine(installDir, "DistFiles"), "icons");
+			// Handle developer machines.
+			var last = Path.GetFileName(installDir).ToLowerInvariant();
+			if (last == "debug" || last == "release")
+			{
+				var parent = Path.GetDirectoryName(installDir);
+				if (parent != null)
+				{
+					last = Path.GetFileName(parent).ToLowerInvariant();
+					if (last == "output")
+					{
+						var grandparent = Path.GetDirectoryName(parent);
+						if (grandparent != null)
+							iconDir = Path.Combine(Path.Combine(grandparent, "DistFiles"), "icons");
+					}
+				}
+			}
```

Comment:
I think you want something like FileLocator.GetDirectoryDistributedWithApplication("icons")